src/middlewared/middlewared/plugins/pool.py

```diff
@@ -4182,6 +4182,131 @@ def __handle_zfs_set_property_error(self, e, properties_definitions):
             raise CallError(pjob.error)
         return data
 
+    @accepts(
+        Str('id', required=True),
+        Str('type', enum=['USER', 'GROUP']),
+        Ref('query-filters'),
+        Ref('query-options'),
+    )
+    @item_method
+    async def get_userquota(self, id, type, filters, options):
+        """
+        Returns a list of the specified `type` of  quotas on the ZFS dataset `id`.
+        Supports `query-filters` and `query-options`. used_bytes and used_percentage
+        may not instantly update as space is used.
+
+        Each quota entry has the following fields:
```

Comment:
Avoid "the following" when items are immediately following.
s/the following/these/